.devops/deploy-pipelines.yml

```diff
@@ -28,16 +24,10 @@ pool:
 
 resources:
   repositories:
-    - repository: pagopaCommonsLegacy
-      type: github
-      name: pagopa/azure-pipeline-templates
-      ref: refs/tags/v4
-      endpoint: 'pagopa'
-
     - repository: pagopaCommons
       type: github
       name: pagopa/azure-pipeline-templates
-      ref: refs/tags/v11
+      ref: refs/tags/v14
       endpoint: 'pagopa'
```

Comment:
```suggestion
      endpoint: 'io-azure-devops-github-ro'
```

Reply:
Done in [68d6ddb](https://github.com/pagopa/io-functions-pushnotifications/pull/11/commits/68d6ddbb9ae3649043341177aef41b1219590305)